mmv1/products/metastore/api.yaml

```diff
@@ -0,0 +1,219 @@
+              name: 'dayOfWeek'
+              description: |
+                The day of week, when the window starts.
+              default_value: :DAY_OF_WEEK_UNSPECIFIED
+              values:
+              - :DAY_OF_WEEK_UNSPECIFIED
+              - :MONDAY
+              - :TUESDAY
+              - :WEDNESDAY
+              - :THURSDAY
+              - :FRIDAY
+              - :SATURDAY
+              - :SUNDAY
+      - !ruby/object:Api::Type::NestedObject
+        name: 'metadataManagementActivity'
+        output: true
+        description: |
+         The metadata management activities of the metastore service.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: metadataExports
+            output: true
+            description: |
+              The latest metadata exports of the metastore service.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Time
+                  name: 'startTime'
+                  description: |
+                    The time when the export started..
+                  output: true
+                - !ruby/object:Api::Type::Time
+                    name: 'endTime'
+                    description: |
+                      The time when the export ended.
+                    output: true
+                - !ruby/object:Api::Type::String
+                  name: 'state'
+                  description: |
+                    The current state of the export.
+                  output: true
+                - !ruby/object:Api::Type::String
+                  name: 'destinationGcsUri'
+                  description: |
+                     A Cloud Storage URI of a folder that metadata are exported to, in the form of `gs://<bucket_name>/<path_inside_bucket>/<export_folder>`.
+                  output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'hiveMetastoreConfig'
+        description: |
+         Configuration information specific to running Hive metastore software as the metastore service.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'version'
+            input: true
+            required: true
+            description: |
+              The Hive metastore schema version.
+          - !ruby/object:Api::Type::KeyValuePairs
+            name: 'configOverrides'
+            description: |
+              A mapping of Hive metastore configuration key-value pairs to apply to the Hive metastore (configured in hive-site.xml).
+              The mappings override system defaults (some keys cannot be overridden)
+          - !ruby/object:Api::Type::NestedObject
+            name: 'kerberosConfig'
+            description: |
+              Information used to configure the Hive metastore service as a service principal in a Kerberos realm.
+            properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: 'keytab'
```

Comment:
Should probably be required?

Reply:
I have trouble sometimes making executive decisions on these apis as for what should be required and what is not needed to be required. I'll need to maybe sync up with you for strategies on this. For me it is not exactly clear what a keytab is or a kerberos config. I think gaining more context on these resources is inherently useful. As for the methodology on how exactly to gain this context I sometimes find myself lost.